lessons/Data-Science/CODAP/codap-filtering/langs/en-us/pages/booleans-with-filter-2.adoc

```diff
@@ -0,0 +1,90 @@
+= Booleans and Filters (2)
+
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+/* Cram everything to the top instead of distributing space equally */
+#content { display: block !important; }
+</style>
+++++
+
+== Booleans and Numbers
+
+In the @starter-file{boolean}, open the Transformer called `filter-is-???`, pictured below. For each prompt below, you will select "Edit" in the Transformer, and then enter the specified Boolean expression. (Relevant boxes are highlighted in red in the image on the right.)
+
+[cols="4, 10", frame="none", grid="none", stripes="none"]
+|===
+
+
+|
+@image{../images/filter-is-blank.png, 300}
+
+|
+
+@n Click Edit. Change `Pounds=` so that it says `Pounds=32`. What happened? @fitb{}{}
+
+@fitb{}{}
+
+@vspace{1ex}
+
+@n What would be a good name for a Transformer with the expression `Pounds<32`?
```

Comment:
Make "name" bold, to highlight it as a key component of a transformer. Ditto for Purpose Statement